src/renderer/src/assets/terminal.css

```
@@ -67,6 +67,33 @@
+   its cursor in, and the helper textarea's caret is transparent (above) — so a
+   Hangul syllable had no caret anywhere until it committed. currentColor tracks
+   the overlay's own foreground, which xterm keeps legible against its ground. */
+/* The preedit caret. Geometry and blink come from
+   terminal-ime-preedit-caret.ts, which knows the pane cell size and cursor
+   settings; this only paints. Scoped to .xterm-container, not
+   .pane-manager-root: panes mount as `.xterm-container > .xterm` and nothing
+   carries that older class today, so a rule written under it never matches. */
+.xterm-container .xterm .composition-view .orca-ime-preedit-caret {
+  display: inline-block;
+  vertical-align: top;
+  background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)